bundle/Resources/views/admin/tag/search.html.twig

```diff
@@ -0,0 +1,47 @@
+{% extends eztags_admin.pageLayoutTemplate %}
+
+{% trans_default_domain 'eztags_admin' %}
+
+{% block content %}
+    <h1 class="context-title">{{ 'tag.search.search_tags'|trans }}</h1>
+
+    <div class="panel">
+        {{ form_start(form, {'attr': {'novalidate': 'novalidate'}}) }}
```

Comment:
Why `novalidate` ?

Reply:
HTML5 validation is clunky and different across browsers, and not really needed here. And I have seen it not work correctly many times. For example, try entering **example@example** in an email type input field... But, ok, I'll enable it.

Reply:
That's true, but HTML5 is always better to have even if it is clunky, than not have it at all. Besides, this is a text field, simple not blank validation is for sure useful.

Reply:
Done in https://github.com/netgen/TagsBundle/commit/f4a8ea03a3e319d60e3f799889a2fd0b12598249